subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/initialization/InstantExecutionStartParameter.kt

```diff
@@ -30,25 +31,20 @@ class InstantExecutionStartParameter(
     private val startParameter: StartParameter
 ) {
 
-    val isEnabled: Boolean by unsafeLazy {
-        systemPropertyFlag(SystemProperties.isEnabled)
-    }
+    val isEnabled: Boolean
+        get() = (startParameter as StartParameterInternal).isConfigurationCacheEnabled
 
     val isQuiet: Boolean
         get() = systemPropertyFlag(SystemProperties.isQuiet)
```

Comment:
The internal _quiet_ flag is not ported as a build option in this PR. We only use it internally for testing purpose. I don't think we want to make it public.

Reply:
Moved as a proper build option like the other, but without public command line option, only a gradle or system property.